sdk/src/realtime/composable.ts

```diff
@@ -111,7 +111,12 @@ export function realtime(config: WebSocketConfig = {}) {
 				if (!message) continue;
 
 				if ('type' in message) {
-					if (message['type'] === 'auth' && hasAuth(currentClient)) {
+					if (
+						message['type'] === 'auth' &&
+						'status' in message &&
+						message['status'] === 'error' &&
+						hasAuth(currentClient)
```

Comment:
Should we make it even more specific by checking the error code too?
In an own client implementation I used the following condition:
```js
data.type === 'auth' &&
data.status === 'error' &&
data.error.code === 'TOKEN_EXPIRED'
```